api/compaction_manager.cc

```diff
@@ -61,6 +74,31 @@ void set_compaction_manager(http_context& ctx, routes& r) {
         });
     });
 
+    cm::get_pending_tasks_by_table.set(r, [&ctx] (std::unique_ptr<request> req) {
+        return ctx.db.map_reduce0([&ctx](database& db) {
+            std::unordered_map<std::pair<sstring, sstring>, uint64_t, utils::tuple_hash> tasks;
+            return do_for_each(db.get_column_families(), [&tasks](const std::pair<utils::UUID, seastar::lw_shared_ptr<table>>& i) {
+                table& cf = *i.second.get();
```

Comment:
we need a futurized version of accumulate()

Reply:
map_reduce does not support future reducer, if that critical to this pach, I'll use a different implementation.

Reply:
No, it was just a remark.

Reply:
ok, I have forced-push the other changes